ui/src/app/components/home/home.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="container">
-  <h3>Get started with HarbourBridge</h3>
-  <div class="summary">
+  <h3 class="primary-header">Get started with HarbourBridge</h3>
+  <div class="summary global-font-color">
```

Comment:
We should set the font size to 14 for summary as suggested in the feedback doc

Reply:
Added body-font-size class to the div